package.json

```diff
@@ -20,10 +20,12 @@
     "test-docs": "jsdoc --configure jsdoc_test.json --verbose"
   },
   "license": "MIT",
-  "dependencies": {
+  "optionalDependencies": {
```

Comment:
Moved the dependencies into the optionalDependencies object.

if we want to not install the dependencies we will have to give the flag `--omit=optional` when installing

Reply:
> `npm install --no-optional` should do the same.

This works but is being phased out. It started getting phased out from npm 7. It still works but they want developers to start using --omit=optional

Reply:
The omit flag was introduced because it's more flexible. 

You can do things like

`npm install --omit=dev,optional`

Reply:
We would want to start adding this kind of install command on certain instances. So that if we know we don't need the s3 bucket packages and all of the dev dependencies we would just use one flag instead of 

`npm install --no-optional --omit=dev`